addons/l10n_ch/models/res_bank.py

```diff
@@ -49,6 +75,14 @@ class ResPartnerBank(models.Model):
              "your own account. The client number is mostly 6 numbers without -, while the postal account number can "
              "be e.g. 01-162-8")
 
+    l10n_ch_qr_iban = fields.Char(string='QR-IBAN',
```

Comment:
:warning: This oversimplifies the need of this field.

We need 2 different fields, otherwise we break the reconciliation process.

The same bank account will have an `IBAN` and a `QR-IBAN`.
With the SEPA process the bank will give a CAMT file that includes the IBAN and not the QR-IBAN.
An other option for it to work would be to be able to have multiple bank account on a single journal. And ensure the reconciliation process works.

cc @fclementic2c

Reply:
Ok,I might have missed the `readonly=False`. If we can store an `IBAN` and a `QR-IBAN` on the same `res.partner.bank` we are good.